src/app/guide/_components/GuideDetail.tsx

```diff
@@ -0,0 +1,54 @@
+import Image from 'next/image';
+import SVGTriangle from '@/styles/icons/triangle.svg';
+
+interface ImageWithDescriptionProps {
+  description: string;
+}
+function ImageWithDescription({ description }: ImageWithDescriptionProps) {
+  return (
+    <div className="flex w-full flex-col items-center">
+      <div className="h-12 w-full bg-white" style={{ paddingTop: '100%' }} />
+      <div className="font-primary-darkblue flex flex-row items-center gap-1 pt-2 font-sfpro text-xs font-extrabold">
+        <SVGTriangle />
+        {description}
+      </div>
+    </div>
+  );
+}
+
+interface GuideDetailProps {
+  title: string;
+  children: React.ReactNode;
+  description1: string;
+  description2: string;
+  description3: string;
+  description4: string;
+}
+
+export default function GuideDetail({
+  title,
+  children,
+  description1,
+  description2,
+  description3,
+  description4,
+}: GuideDetailProps) {
+  return (
+    <div className="flex flex-col">
+      <div className="flex flex-row items-center pt-8">
+        <Image src="/profile.png" alt="Sample Image" width={72} height={72} />
+        <div className="pl-4">
+          <div className="text-xl">{title}</div>
+          <div className="font-sfpro text-sm text-white">{children}</div>
+        </div>
+      </div>
+
+      <div className="flex grid grid-cols-2 justify-center gap-x-4 gap-y-4 pt-4">
```

Comment:
```suggestion
      <div className="grid grid-cols-2 justify-center gap-x-4 gap-y-4 pt-4">
```

사소하긴 한데 css에서 사용하지 않게된 css를 지워주는게 css 디버깅시에 중요하게 느껴지더라고요. 지금 회사에서 퍼블리싱을 받아서 사용하는데 실제로 의미 없는 css 코드가 너무 많아서 디버깅할때 좀 어렵더라구요!